canonical-web-and-design/ubuntu.com
# Build page on /kubernetes/resources

## Done

- Build page on `/kubernetes/resources`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Please compare page against [copy doc](https://docs.google.com/document/d/1enaCJ4Vt9a_2_l6xYccnZ-fBVUejPSmfMi8LP2aDA9c/edit)
- Demo at https://ubuntu-com-11544.demos.haus/kubernetes/resources

## Issue / Card

Fixes [#5089](https://github.com/canonical-web-and-design/web-squad/issues/5089)




## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)
